src/content/reference/manifest.mdx

```diff
@@ -214,6 +214,16 @@ deploy:
   compose: docker-compose.yml
 ```
 
+You can share environment variables between steps by adding them to `$OKTETO_ENV`:
```

Comment:
Will this work across dependencies? IF so, can we add it to the docs?

Reply:
it will only works across deploy commands